plugins/colorbutton/plugin.js

```diff
@@ -210,14 +210,14 @@ CKEDITOR.plugins.add( 'colorbutton', {
 				if ( color == '?' ) {
 					editor.getColorFromDialog( function( color ) {
 						if ( color ) {
-							return applyColor( color );
+							return applyOrRemoveColor( color );
 						}
 					} );
 				} else {
-					return applyColor( color );
+					return applyOrRemoveColor( color );
 				}
 
-				function applyColor( color ) {
+				function applyOrRemoveColor( color ) {
```

Comment:
`applyOrRemoveColor` is not a good name for the method. From what I can see is the method will unset the style if it gets "falsy" `color` value. In that case name it simply `setColor`.

Avoid names without clear meaning / ambiguous purpose.